Sources/XcodeProj/Scheme/XCScheme+TestableReference.swift

```diff
@@ -6,7 +6,7 @@ public extension XCScheme {
         // MARK: - Attributes
 
         public var skipped: Bool
-        public var parallelizable: Bool
+        public var parallelization: Parallelization
```

Comment:
I'd probably keep the name of this property `parallelizable` to keep aligned with the `pbxproj` format.